### PR TITLE
Update Brazilian's Portuguese translation

### PR DESCRIPTION
**Pull request by samueloph**
Adds https://github.com/pkp/pkp-lib/commit/13baf96a2f7b173487fce7955d36a0ef67448557 changes.

### CODE REVIEW

**Comment by samueloph** [comment]
Should I make another PR against the master branch or you are going to cherry pick it?

**Comment by samueloph** [comment]
This missing string is causing ojs to drop untranslated text onto admin config page.

Is there someone working on accepting my PR?

screenshot from 2015-11-10 14-57-18

File: locale/pt_BR/admin.xml [hunk 1]
```diff
@@ -93,8 +93,10 @@
 	<message key="admin.scheduledTask.startTime">Tarefa iniciada.</message>
 	<message key="admin.scheduledTask.stopTime">Tarefa parada.</message>
 	<message key="admin.scheduledTask.noLog">Tarefa não produziu registros (logs).</message>
-	<message key="admin.scheduledTask.openAccessNotification">Notificação de acesso aberto</message>
```
Comment (beghelli) [inline locale/pt_BR/admin.xml]: why you removed this one? It's being used by classes/tasks/OpenAccessNotification.inc.php

**Comment by mtub** [comment]
I am sorry, @samueloph - thanks for the reminder. I will merge the PR and will port to master.

File: locale/pt_BR/admin.xml [hunk 2]
```diff
@@ -93,8 +93,10 @@
...
 	<message key="admin.scheduledTask.reviewReminder">Lembrete de avaliador</message>
 	<message key="admin.scheduledTask.subscriptionExpiryReminder">Lembrete de expiração de assinatura.</message>
 	<message key="admin.fileLoader.fileProcessed">O arquivo {$filename} foi processado e arquivado.</message>
+	<message key="admin.scheduledTask.downloadLog">Sua instalação OJS automaticamente executou e finalizou esta tarefa, você pode baixar os registros (logs) aqui: {$url}</message>
```
Comment (beghelli) [inline locale/pt_BR/admin.xml]: Ficaria melhor "executou automaticamente", eu acho. ;)

File: locale/pt_BR/admin.xml (hunk 1 above)
Comment (samueloph) [inline locale/pt_BR/admin.xml]: Wait what? It got removed in https://github.com/pkp/pkp-lib/commit/13baf96a2f7b173487fce7955d36a0ef67448557.

I'm gonna check to see if anybody add it back.

Comment (beghelli) [inline locale/pt_BR/admin.xml]: You're right, I did it. Don't worry, it's already merged in. I will take a look why it's still being used. Thanks for your contribution. :)

File: locale/pt_BR/admin.xml (hunk 2 above)
Comment (samueloph) [inline locale/pt_BR/admin.xml]: Realmente, devo fazer outro PR apenas com esta alteração?

**Comment by mtub** [comment]
@beghelli is right, the string should stay in the file. Can you update your fork and send another PR, @samueloph? Otherwise I can add the line again.

File: locale/pt_BR/admin.xml (hunk 2 above)
Comment (beghelli) [inline locale/pt_BR/admin.xml]: Pode deixar, eu mudo. Brigado. ;)

**Pull request merged**